src/raster/r.damflood/main.c

```diff
@@ -52,7 +52,7 @@
 
 
 //#define hmin 0.01
-/* lo calcolo dopo in funzione della velocita' massima 
+/* Calculated later as a function of maximum velocity
```

Comment:
Having these three lines commented here, saying something that is added in context elsewhere seems weird, but I didn't  dare to change it